### PR TITLE
Add AITRIAGE to Loki scripts

### PR DESCRIPTION
**Pull request by vrutkovs**
These scripts are handy to analyze assisted-installer failure logs, running Loki-Grafana-Promtail stack to query logs from multiple sources

### CODE REVIEW

**Comment by osherdp** [comment]
I'll take a look after lunch

File: local/aitriage-to-loki/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+AITRIAGE logs to Loki
+====
+
+These two script help with triaging assisted-installer failures. Looking into separate files makes it complicated to correlate particular events from multiple sources - i.e. bootkube log and infraenv/cluster events.
```
Comment (osherdp) [inline local/aitriage-to-loki/README.md:4]: nit:
```suggestion
These two scripts help with triaging assisted-installer failures. Looking into separate files makes it complicated to correlate particular events from multiple sources - i.e. bootkube log and infraenv/cluster events.
```

**Review by osherdp — CHANGES_REQUESTED** [review]
Very nice!
just one suggestion - I would replace the ``local/`` dir with ``triage/`` so that we can start categorizing scripts between the different roles: ``ci``/``triage``/``test(?)``

File: local/aitriage-to-loki/README.md (hunk 1 above)
Comment (vrutkovs) [reply local/aitriage-to-loki/README.md]: Good idea, amended the commit

**Review by osherdp — APPROVED** [review]

**Pull request merged**